### PR TITLE
[LJPS-58] View courses related to a skill

### PR DESCRIPTION
**Pull request by lanceljr**
## Context

JIRA ticket: https://g4t7.atlassian.net/jira/software/c/projects/LJPS/issues/LJPS-58

## Checklist

- [x] My branch is up-to-date with the `main` branch
- [x] Unit test written
- [x] Acceptance Criteria considered and implemented
- [x] Automated tests are passing

### CODE REVIEW

**Review by bongjiajun — APPROVED** [review]
Nice!

File: ljps-frontend/pages/learning-journey/view-skills/view-courses/index.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ export default function ViewCourses() {
   const [roleName, setRoleName] = useState("");
 
   useEffect(() => {
-    if (skillID && roleID) {
-      const url = `${process.env.NEXT_PUBLIC_BACKEND}/api/skills/${skillID}`
+    if (skills && roleID) {
+      const url = `${process.env.NEXT_PUBLIC_BACKEND}/api/skills/multiple/${skills}`
```
Comment (khyechun) [inline ljps-frontend/pages/learning-journey/view-skills/view-courses/index.js:22]: Is this endpoint changed?

**Review by khyechun — APPROVED** [review]
Looks good : )

File: ljps-frontend/pages/learning-journey/view-skills/view-courses/index.js (hunk 1 above)
Comment (lanceljr) [reply ljps-frontend/pages/learning-journey/view-skills/view-courses/index.js:22]: Hi Khye Chun, there were no changes to the endpoint. I simply reverted the api call on the frontend, cheers!

**Pull request merged**